packages/api/config/lambdas.yml

```diff
@@ -90,3 +90,14 @@ InRegionS3Policy:
   timeout: 300
   source: 'node_modules/@cumulus/api/dist/'
 
+EmsReport:
+  handler: index.emsReport
+  timeout: 300
+  memory: 512
+  source: 'node_modules/@cumulus/api/dist/'
+  useElasticSearch: '{{es.name}}'
+  envs:
+    bucket: '{{buckets.internal}}'
+    ems_provider: '{{ems.provider}}'
+    stackName: '{{stackName}}'
```

Comment:
stackName is automatically added to all the lambda functions. we can remove this one

Reply:
thanks. removed stackName.